diff --git a/node_modules/@libp2p/interface-internal/dist/src/connection-manager/index.d.ts b/node_modules/@libp2p/interface-internal/dist/src/connection-manager/index.d.ts
index 14d3f8c..8e6d2a9 100644
--- a/node_modules/@libp2p/interface-internal/dist/src/connection-manager/index.d.ts
+++ b/node_modules/@libp2p/interface-internal/dist/src/connection-manager/index.d.ts
@@ -3,6 +3,11 @@ import type { Connection, MultiaddrConnection } from '@libp2p/interface/connecti
 import type { PeerId } from '@libp2p/interface/peer-id';
 import type { PeerMap } from '@libp2p/peer-collections';
 import type { Multiaddr } from '@multiformats/multiaddr';
+
+export interface OpenConnectionOptions extends AbortOptions {
+  priority?: number
+}
+
 export interface ConnectionManager {
     /**
      * Return connections, optionally filtering by a PeerId
@@ -34,7 +39,7 @@ export interface ConnectionManager {
      * const connection = await libp2p.connectionManager.openConnection(peerId)
      * ```
      */
-    openConnection: (peer: PeerId | Multiaddr | Multiaddr[], options?: AbortOptions) => Promise<Connection>;
+    openConnection: (peer: PeerId | Multiaddr | Multiaddr[], options?: OpenConnectionOptions) => Promise<Connection>;
     /**
      * Close our connections to a peer
      */
